common/pom.xml

```diff
@@ -37,6 +67,10 @@
       <scope>compile</scope> <!-- override the 'test' scope defined at parent pom.xml -->
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jctools</groupId>
+      <artifactId>jctools-core</artifactId>
+    </dependency>
```

Comment:
I have always been told that shade plugin is very shady, it does things beyond anyone's comprehension, you know when you tell to non IT users how something works and you just say: "magic"